packages/rulesets/src/spectral/test/exception-mandate-for-tenant-level-api-path.test.ts

```diff
@@ -0,0 +1,114 @@
+import { Spectral } from "@stoplight/spectral-core"
+import linterForRule from "./utils"
+
+const ERROR_MESSAGE = "is a tenant level api and will need exception from PAS team."
+
+let linter: Spectral
+
+beforeAll(async () => {
+  linter = await linterForRule("ExceptionMandateForTenantLevelApiPath")
+  return linter
+})
+
+test("ExceptionMandateForTenantLevelApiPath should find no errors for non tenant level api paths and operations api-path", () => {
+  const oasDoc = {
+    swagger: "2.0",
+    paths: {
+      "/providers/Microsoft.Music/operations": {
+        put: {
```

Comment:
> put: { [](http://example.com/codeflow?start=7&length=7)

nit: Typically operations only has a get. We can still keep the condition to exclude operations even though the filter for put will filter it out.. no action required on this comment :) ..